source/sass_api_reference/index.html.md.erb

```diff
@@ -0,0 +1,85 @@
+---
+title: Sass API Reference
+weight: 65
+---
+
+# Sass API Reference
+
+<%# Group the docs by their 'group' %>
+<% groupedDocs = data.sassdoc.group_by{ |doc| doc.group.first } %>
+<% groupedDocs.each do |index, group| %>
+
+## <%= index == 'undefined' ? 'General' : index.gsub('/', ' / ').titlecase %>
+
+    <%# Remove private entries %>
+    <% docs = group.select{ |doc| doc.access == 'public' } %>
+    <%# Remove vendored files %>
+    <% docs = docs.select{ |doc| !doc.file.path.start_with?('vendor') } %>
+    <% docs.each do |doc| %>
+
+### <%= doc.context.type == 'variable' ? '$' : ''  %><%= doc.context.name %>
+
+<%= doc.description %>
+
+        <% if doc.parameter %>
+            <%= partial(:parameters_table, locals: { doc: doc }) %>
+        <% end %>
+
+#### Usage
+
+        <% if doc.parameter %>
+            <% inline_parameters = capture do %>
+                <% doc.parameter.each_with_index do |param, index| -%>
+<%= partial(:inline_parameters, locals: { doc: doc, param: param, index: index }) -%>
+                <% end %>
+            <% end %>
+            <% inline_parameters = "(#{inline_parameters})" %>
+        <% end %>
+
+        <% if doc.context.type == 'mixin' %>
+            <% acceptsContent = doc.context.code.include?('@content') %>
+            <% trailingCode = capture do %><% if acceptsContent %> {
+    .app-selector {
+        // ...
+    }
+}<% else %>;<% end -%>
+            <% end %>
+
+```scss
+@include <%= doc.context.name %><%= inline_parameters %><%= trailingCode -%>
+```
+        <% end %>
+
+        <% if doc.context.type == 'function' %>
+```scss
+$result: <%= doc.context.name %><%= inline_parameters -%>;
+```
+        <% end %>
+
+        <% if doc.context.type == 'variable' %>
+```scss
+$<%= doc.type.downcase %>: $<%= doc.context.name %>;
+```
+        <% end %>
+
+        <% if doc.example %>
+#### Example
```

Comment:
I don't like how the markdown needs to be indented differently to the rest of the file for it to render properly, makes it hard to read...

Reply:
https://github.com/alphagov/tech-docs-gem/pull/170